library/ssl_tls13_keys.c

```diff
@@ -1010,16 +1010,16 @@ int mbedtls_ssl_tls13_generate_handshake_keys( mbedtls_ssl_context *ssl,
                 MBEDTLS_SSL_KEY_EXPORT_TLS1_3_CLIENT_HANDSHAKE_TRAFFIC_SECRET,
                 tls13_hs_secrets->client_handshake_traffic_secret,
                 md_size,
-                handshake->randbytes + 32,
                 handshake->randbytes,
+                handshake->randbytes + 32,
```

Comment:
While at it we have a mnemonic now for the 32 here: MBEDTLS_CLIENT_HELLO_RANDOM_LEN.
Thus
```suggestion
                handshake->randbytes + MBEDTLS_CLIENT_HELLO_RANDOM_LEN,
```
would be better here I'd say.
```